cli/internal/cmdutil/errors.go

```diff
@@ -0,0 +1,14 @@
+package cmdutil
+
+// Error is a specific error that is returned by the command to specify the exit code
+type Error struct {
+	ExitCode int
+	Err      error
+}
+
+func (e *Error) Error() string { return e.Err.Error() }
+
+// BasicError is an empty error that is returned by the command
+type BasicError struct{}
+
+func (e *BasicError) Error() string { return "basic error" }
```

Comment:
I'd rather not have a dummy value like this if we can avoid it. I mentioned above, but I think if we split the logging of an error and returning an error, we can get away without needing this.

Reply:
We can't split the logging of an error and returning an error, we need to return some form of an error to at least exit with exit code 1, and using `ExitCodeError` would take up more space than an empty error. That's why I was so adamant on using the new UI.